dashboard/app/views/levels/_single_multi.html.haml

```diff
@@ -90,14 +90,11 @@
 -# Don't operate if we're a LevelGroup with a /level/ URL.
 - if standalone || @script_level
   :javascript
-
-    $(document).ready(function() {
-      window["level_#{index}"] = new Multi(
-        "level_#{index}",
-        #{level.id},
-        #{standalone},
-        "#{data['type']}" == "multi2" ? 2 : 1,
-        #{data['answers'].map {|answer| answer['correct']}},
-        #{last_attempt.to_json}
-        );
-      });
+    window.levelGroup.results[#{level.id}] = new Multi(
```

Comment:
It seems weird to be assigning a Multi object into something called results, doesn't it?